### PR TITLE
Elasticsearch: The Definitive Guide is no longer maintained

### PR DESCRIPTION
**Pull request by jrodewig**
We no longer maintain Elasticsearch: The Definitive Guide or this repo. These docs only
cover the 1.x and 2.x versions of Elasticsearch, which have passed their EOL
dates.

Those interested in the latest info should use the [current Elasticsearch
docs][0] instead.

I'll backport the changes to the 2.x and 1.x branches as well.

### Changes

* Updates the page header and README to clearly state the docs are no longer maintained.
* Updates the contribution guidelines to discourage pull request and issues.
* Removes a section of the title page for contributions.

### Preview
Screen Shot 2021-09-17 at 6 31 23 PM

[0]: https://www.elastic.co/guide/en/elasticsearch/reference/current/index.html

### CODE REVIEW

File: page_header.html [hunk 1]
```diff
@@ -1,4 +1,12 @@
-This information may not apply to the latest version of Elasticsearch.
-For the most up to date information, see the current version of the
-<a href="https://www.elastic.co/guide/en/elasticsearch/reference/current/index.html">
-Elasticsearch Reference</a>.
+<p>
+  <strong>WARNING</strong>: This documentation cover Elasticsearch 2.x. The 2.x
```
Comment (jrodewig) [inline page_header.html:2]: > This documentation cover Elasticsearch 2.x.

I'll remove this sentence on the 2.x and 1.x backports, where the version is more obvious.

File: page_header.html [hunk 2]
```diff
@@ -1,4 +1,12 @@
-This information may not apply to the latest version of Elasticsearch.
-For the most up to date information, see the current version of the
-<a href="https://www.elastic.co/guide/en/elasticsearch/reference/current/index.html">
-Elasticsearch Reference</a>.
+<p>
+  <strong>WARNING</strong>: This documentation covers Elasticsearch 2.x. The 2.x
```
Comment (jrodewig) [inline page_header.html:2]: > This documentation covers Elasticsearch 2.x.

I'll remove this sentence in the 2.x and 1.x backports, where the version is more obvious.

File: README.md [hunk 3]
```diff
@@ -26,13 +30,16 @@ The Definitive Guide is written in Asciidoc and the docs repo also contains a [s
...
+* The [`2.x` and `master` branches](https://github.com/elastic/elasticsearch-definitive-guide/tree/2.x) apply to Elasticsearch 2.x
 
 ## Contributing
 
-Before contributing a change please read our [contribution guide](CONTRIBUTING.md).
+This repository is no longer maintained. Pull requests and issues will not be
+addressed.
+
+To contribute to the current Elasticsearch docs, refer to the [Elasticsearch
+repo](https://github.com/elastic/elasticsearch/).
```
Comment (xeraa) [inline README.md:42]: Very minor nitpick: Should we spell out "repo" for consistency?

**Review by xeraa — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (jrodewig) [reply README.md]: Yes. Thanks!

**Review by gtback — APPROVED** [review]
LGTM!

**Comment by jrodewig** [comment]
### Backports
2.x: https://github.com/elastic/elasticsearch-definitive-guide/commit/07acfc9d998108ec22d9d36b4777d228de64fe0c
1.x: https://github.com/elastic/elasticsearch-definitive-guide/commit/ab5e9555ef7a18dd9eb917e4ddfe4557213b5ef3

**Pull request merged**